Procfile

```diff
@@ -0,0 +1 @@
+web: ./node_modules/.bin/next start
```

Comment:
Does this work without a $PORT value?

Reply:
Yes, I have deployed it on a heroku server to test with the `procfile`, It successfully build and deployed without any `$PORT` 
related issue. All the issue I found was with the '.env' vars as I provided some random data for it. Is it neccessary to put that `$PORT` value for deployment?  
![image](https://user-images.githubusercontent.com/45600689/194011779-9c162643-630b-4cd0-bdd4-b724130d1c3c.png)